CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ $ TEST_DEBUG=true make test
 To test the code coverage, use:
 
 ```sh
-$ make test-cov
```

Comment:
@zjmiller up for sending a PR that updates CONTRIBUTING.md to simply call `./scripts/test-cov.sh` directly?